wrappers/java/src/test/java/org/hyperledger/indy/sdk/utils/InitHelper.java

```diff
@@ -3,12 +3,10 @@
 
 import org.hyperledger.indy.sdk.LibIndy;
 
-import java.io.File;
-
 public class InitHelper {
 	public static void init() {
 
-		if (!LibIndy.isInitialized()) LibIndy.init(new File("./lib/libindy"));
+		if (!LibIndy.isInitialized()) LibIndy.init("./lib/");
```

Comment:
It doesn't look for libindy inside the jar.

Calling `LibIndy.init()` will load libindy from system LD path. <-- default
Calling `LibIndy.init(new File("./lib/libindy.so"))` will load libindy from a given file path. <-- not platform independent.
Calling `LibIndy.init("./lib/")` will load libindy from a given directory path. <-- useful for testing, this is new.